internal/clientconn/conn.go

```diff
@@ -250,3 +247,59 @@ func (c *conn) run(ctx context.Context) (err error) {
 		}
 	}
 }
+
+// route routes to common.Router and measures all the result.
+func (c *conn) route(h common.Handler, ctx context.Context, reqHeader *wire.MsgHeader, reqBody wire.MsgBody,
```

Comment:
Looks like parameter `h common.Handler` is unused. Should we remove it?

Reply:
It's used on the line 290